### PR TITLE
Add Huawei Maimang 5 / Nova (Plus) / G9 (Plus)

### PR DESCRIPTION
**Pull request by bzy-080408**
support devices:
Huawei CAN-AL00
Huawei CAN-L01
Huawei CAN-L02
Huawei CAN-L03
Huawei CAN-TL00
Huawei MLA-AL00
Huawei MLA-L01
Huawei MLA-L02
Huawei MLA-L03
Huawei MLA-TL00

### CODE REVIEW

**Comment by bzy-080408** [comment]
sorry I forget it ...

File: dts/msm8953_huawei_common.dtsi [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    };
+    qcom,mdss_dsi_tianma_fic8736_5p5_1080p_cmd{
+      compatible = "huawei,tianma_fic8736_5p5_1080p_cmd";
+    };
+    qcom,mdss_dsi_tianma_focal8716_5p5_1080p_video{
+      compatible = "huawei,tianma_focal8716_5p5_1080p_video";
+    };
+    qcom,mdss_dsi_tianma_otm1906c_5p5_1080p_cmd{
+      compatible = "huawei,tianma_otm1906c_5p5_1080p_cmd";
+    };
```
Comment (z3ntu) [inline dts/msm8953_huawei_common.dtsi:36]: Fix all the indent in this file

File: dts/msm8953_huawei_common.dtsi [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2023, bzy-080408<Bzy080408@outlook.com>
+
+/ {
+  model = "huawei big error";
```
Comment (z3ntu) [inline dts/msm8953_huawei_common.dtsi:5]: ?

**Comment by z3ntu** [comment]
What are these tens of random model numbers? Are these actual phones? I doubt Huawei sells something branded as e.g. "Huawei MLA-L01"

**Comment by bzy-080408** [comment]
> What are these tens of random model numbers? Are these actual phones? I doubt Huawei sells something branded as e.g. "Huawei MLA-L01"

Huawei has two ways to name devices, one is the code name（e.g. Huawei Nova Plus;Huawei Maimang5) and the other is the model (e.g. MLA-AL00;CAN-L01)

**Comment by z3ntu** [comment]
@bzy-080408 Then please add proper names also to both README and the dts files.

**Comment by bzy-080408** [comment]
but one model of huawei device have more than two board-id and need more than two device-tree ,so how to name the device tree

**Comment by bzy-080408** [comment]
and the system showed the model name

File: dts/msm8953-huawei-milan.dts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	compatible = "huawei,sdm625", "qcom,msm8953", "lk2nd,device";
+	qcom,msm-id = <293 0x0>;
+	qcom,board-id = <8000 0>, <8001 0>, <8002 0>, <8003 0>, <8016 0>,
+			<8017 0>, <8018 0>, <8019 0>, <8032 0>, <8033 0>,
+			<8035 0>, <8048 0>, <8049 0>, <8050 0>, <8051 0>,
+			<8066 0>, <8064 0>, <8067 0>, <8080 0>, <8082 0>,
+			<8083 0>, <8098 0>, <8099 0>, <8114 0>, <8130 0>;
+
+	panel {
+		compatible = "huawei,sdm625-panel";
```
Comment (z3ntu) [inline dts/msm8953-huawei-milan.dts:19]: dunno what convention we use for panel compatibles, we might want to change that

Comment (z3ntu) [reply dts/msm8953-huawei-milan.dts:19]: changed it to `huawei,milan-panel` to align with other devices

**Review by z3ntu — APPROVED** [review]
Thanks!

**Pull request merged**